### DISCUSSION TITLE
Unable to use plotly js in Gatsby.js

### DISCUSSION THREAD

**Discussion by lakshmishreea122003** [gatsbyjs/gatsby#38906] [Help]
I am unable to figure out how to use plotly in my gatsby.js project.
I get build error when I use import * as px from plotly.express .
Can you please guide me as to how I can use plotly in gatsby. js

**Comment by lakshmishreea122003** [comment]
I am trying to make a gatsby project that shows dashboards. Thus using Plotly js is important. So kindly consider to guide me

**Comment by violy** [comment]
Hi Lakshmi, 

I discover Plotly with your post ! But I'm trying to make a response…

If you are using [the `plotly.js` library](https://github.com/plotly/plotly.js) the readme says, after installation of `npm i --save plotly.js-dist-min` : 
```js
// ES6 module
import Plotly from 'plotly.js-dist-min'
```

Also, you are writing about [`plotly.express` … but it seems a Python library](https://plotly.com/python/plotly-express/). Gatsby is Javascript / TypeScript framework …

Arthur.
